coverage.gradle

```diff
@@ -0,0 +1,97 @@
+def excludes = [
```

Comment:
we'll probably want to sync with the team to know which files to exclude from the code coverage, could keep this as an `def excludes = []` for the time being as most of these are project specific

Reply:
Going to exclude nothing so we can get the runs going; then allow people to iterate on the exclusions.